README.md

```diff
@@ -1,6 +1,48 @@
 # Easier68k [![Build Status](https://travis-ci.org/Chris-Johnston/Easier68k.svg?branch=master)](https://travis-ci.org/Chris-Johnston/Easier68k)
 
-A python package based assmebler, disassembler and simulator of the Motorola 68k CPU Architecture.
+Easier68k is a python library that assembles to 68k and simulates the Motorola 68k CPU Architecture.
+It is built as a library so anyone can create a front end for it. Currently this repository includes a CLI and a GUI is in the works.
+
+NOTE: Easier68k is currently under heavy production and is not stable.
+
+### Motivation
+
+There are very few 68K simulators and assemblers. The only one which is still maintained and complete is Easy68k.
+We realized we could do better:
+* Easy68k is not cross platform, but Easier68k is
+* Easy68k is not modular and enforces you to use its editor and GUI, but Easier68k is built as a modular library so it can fit in anyones personal setup
+* Easy68k is buggy, but Easier68k has a focus on correctness by using unit tests and integration tests for everything.
+* Easy68k is GPL licensed, but Easier68k is MIT licensed
+
+
+### How to Use?
+
+If you want to use this as a library to either develop a front end or automate some special tasks then simply import the
+package in your project and start coding.
+
+If you want to use this to assemble and simulate some 68k assembly then keep on reading.
+
+Currently there is only one front end available for easier68k which is the easier68k-cli which comes in this Github repository.
```

Comment:
I would replace the wording "How to Use?" with just "CLI", since this section doesn't explain how to install and import the package, where elsewhere it does.